libnetwork/drivers/bridge/bridge_store.go

```diff
@@ -137,6 +138,7 @@ func (ncfg *networkConfiguration) MarshalJSON() ([]byte, error) {
 	nMap["GwModeIPv4"] = ncfg.GwModeIPv4
 	nMap["GwModeIPv6"] = ncfg.GwModeIPv6
 	nMap["EnableICC"] = ncfg.EnableICC
+	nMap["TrustedHostInterfaces"] = strings.Join(ncfg.TrustedHostInterfaces, " ")
```

Comment:
Why not simply marshal the slice of strings as a JSON array of strings? The persistence of bridge network configurations is so far removed from parsing the netlabels that I am unconvinced that there would be any maintenance or code readability advantage to using the same serialization scheme in the persistence layer. Marshaling to a JSON array has the advantage of being compatible with the default un/marshalling of a Go `[]string` struct field. With a stringly-typed list we would have to define a custom type that implements `json.Marshaler` and `json.Unmarshaler` in order to un/marshal directly to/from a struct.

Reply:
> the other way to be to unmarshal by type asserting the value to `[]interface{}`, then iterate over that and type assert the elements to strings.

Fair point. Though just for completeness' sake, we do have a `"./internal/sliceutil".Map` function which makes that operation not much more verbose than a `strings.FieldsFunc` call.
```go
sliceutil.Map(v, func(a any) string { s, _ := a.(string); return s })
```